src/components/Incorporation/SummaryDefineCompany.vue

```diff
@@ -1,11 +1,11 @@
 <template>
-  <v-card flat class="rounded-0">
-    <div class="define-company-header review-header">
+  <v-card flat id="summary-define-company">
+    <div class="define-company-header">
       <v-icon color="appDkBlue">mdi-domain</v-icon>
       <label class="define-company-title pl-2"><strong>Your {{ getCompanyDisplayName }}</strong></label>
     </div>
 
-    <section :class="{ 'invalid-section': !isDefineCompanyValid }">
+    <section :class="{ 'invalid-section rounded-bl-0': !isDefineCompanyValid }">
```

Comment:
When the left red bar is displaying, remove bottom-left rounding so bar doesn't get rounded (which looks odd).

Reply:
No longer needed. This base class into play now:

```
.invalid-section {
  border-left: 3px solid $app-red !important;
  border-top-left-radius: 0 !important;
  border-bottom-left-radius: 0 !important;
```